### PR TITLE
.Atan() entry branch

### PR DESCRIPTION
**Pull request by OCollis**
### Description

New .md file containing an entry for the C# math function .Atan() at ` docs/content/c-sharp/concepts/math-functions/terms/atan/atan.md`

### Type of Change

- Adding a new entry
- 
### Checklist

- [ x ] All writings are my own.
- [ x ] My entry follows the Codecademy Docs style guide.
- [ x ] My changes generate no new warnings.
- [ x ] I have performed a self-review of my own writing and code.
- [ x ] I have checked my entry and corrected any misspellings.
- [ x ] I have made corresponding changes to the documentation if needed.
- [ x ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ x ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ x ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: .vs/ProjectSettings.json
Comment (SSwiniarski) [inline .vs/ProjectSettings.json:1]: Remove this file

File: .vs/VSWorkspaceState.json
Comment (SSwiniarski) [inline .vs/VSWorkspaceState.json:1]: Remove this file

File: .vs/docs/FileContentIndex/021f9686-ce2d-49d5-b351-bb365cf49ef7.vsidx
Comment (SSwiniarski) [inline .vs/docs/FileContentIndex/021f9686-ce2d-49d5-b351-bb365cf49ef7.vsidx:1]: Remove this file

File: .vs/docs/FileContentIndex/7055da89-0860-4422-9ebe-a06db798e266.vsidx
Comment (SSwiniarski) [inline .vs/docs/FileContentIndex/7055da89-0860-4422-9ebe-a06db798e266.vsidx:1]: remove this file

File: .vs/docs/FileContentIndex/70f08246-1abd-4b3a-8a52-7cd9cfff4c09.vsidx
Comment (SSwiniarski) [inline .vs/docs/FileContentIndex/70f08246-1abd-4b3a-8a52-7cd9cfff4c09.vsidx:1]: Remove this file

File: .vs/docs/FileContentIndex/read.lock
Comment (SSwiniarski) [inline .vs/docs/FileContentIndex/read.lock:1]: Remove this file

File: .vs/docs/v17/.wsuo
Comment (SSwiniarski) [inline .vs/docs/v17/.wsuo:1]: Remove this file

File: .vs/docs/v17/workspaceFileList.bin
Comment (SSwiniarski) [inline .vs/docs/v17/workspaceFileList.bin:1]: Remove this file

File: .vs/slnx.sqlite
Comment (SSwiniarski) [inline .vs/slnx.sqlite:1]: remove this file

File: content/c-sharp/concepts/math-functions/terms/atan/atan.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/math-functions/terms/atan/atan.md:2]:
```suggestion
```
Remove extra line break.

File: content/c-sharp/concepts/math-functions/terms/atan/atan.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+**Math.Atan()** is a static method that calculates the inverse tangent of a given number, in radians.
+
+## Syntax
+
+```pseudo
+Math.Atan(x);
+```
+- The `Math.Atan()` method takes one double as an argument and returns a double.
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/math-functions/terms/atan/atan.md:23]:
```suggestion

- The `Math.Atan()` method takes one double as an argument and returns a double.
```

Add line break.

File: content/c-sharp/concepts/math-functions/terms/atan/atan.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+}
+```
+
+This example will print the following output:
+
+```shell
+0.6435011087932844 
+```
+
+The example below shows the code in action:
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/math-functions/terms/atan/atan.md:50]: The codebyte example:
1. Should have a header `## Codebyte Example`
2. Should be different from the example already given.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@OCollis I did a first review and left some comments. Most important is that your PR contains a number of extra files that need to be removed.

**Comment by OCollis** [comment]
I have removed the unessesary files and updated `content/c-sharp/concepts/math-functions/terms/atan/atan.md` based on SSwiniarski's feedback

File: content/c-sharp/concepts/math-functions/terms/atan/atan.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
+---
+
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/math-functions/terms/atan/atan.md:2]:
```suggestion
```
You still need to remove the extra line break at the start of the metadata.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@OCollis, two more comments.

File: content/c-sharp/concepts/math-functions/terms/atan/atan.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+## Syntax
+
+```pseudo
+Math.Atan(x);
+```
+
+- The `Math.Atan()` method takes one double as an argument and returns a double.
+- Using this method requires the `System` namespace.
+
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/math-functions/terms/atan/atan.md:24]:
```suggestion
```
Remove extra line break.

**Review by SSwiniarski — APPROVED** [review]
@OCollis aside from one more line break, this is ready for a second review.

File: content/c-sharp/concepts/math-functions/terms/atan/atan.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+Title: '.Atan()'
+Description: 'Returns the inverse tangent of the argument.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Arithmetic'
+  - 'Methods'
+  - 'Numbers'
+CatalogContent:
+  - 'Learn c-sharp'
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/math-functions/terms/atan/atan.md:11]:
```suggestion
  - 'learn-c-sharp'
```

File: content/c-sharp/concepts/math-functions/terms/atan/atan.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+}
+```
+
+This example will print the following output:
+
+```shell
+0.6435011087932844 
+```
+
+## Codebyte Example
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/math-functions/terms/atan/atan.md:48]: Generally, for simple methods, one example is sufficient unless there is some valuable additional functionality demonstrated by a second example. Here I think we should remove one (either example can stay on its own).

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@OCollis Overall looks good. Just a couple of minor edits.

**Review by caupolicandiaz — APPROVED** [review]
@OCollis All set. I did add a quick edit, the catalogContent item was still missing a dash.

**Comment by SSwiniarski** [comment]
@OCollis congrats on the new entry. Live link is here: https://www.codecademy.com/resources/docs/c-sharp/math-functions/atan

You're currently listed as an anonymous contributor. If you want to be credited follow the instructions below to add your GitHub username to your Codecademy profile. If not, there is nothing more you need to do.

image

To add your GitHub username to your Codecademy profile:

1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:

image

2.) Then log out and log back into Codecademy using your GitHub login:

image

**Pull request merged**